README.md

```diff
@@ -1,2 +1,171 @@
+in [talk.pdf](talk.pdf) as well as Julia scripts in the [examples](examples/)
+directory to locally create results of mesh generation with
+[HOHQMesh.jl](https://github.com/trixi-framework/HOHQMesh.jl)
+and [Trixi.jl](https://github.com/trixi-framework/Trixi.jl) simulations
+presented in the talk.
+
+To reduce the size of the file for the [talk.pdf](talk.pdf), the video is not
+embedded. Instead, the video `cylinder_flow_sine_walls.mp4` shown in the
+presentation is located in [media](media/).
+
+
+## Abstract
+
+We present a Julia toolchain for the adaptive simulation of hyperbolic PDEs
+such as flow equations on complex domains. It begins with HOHQMesh.jl to
+create a curved, unstructured mesh. This mesh is then used in Trixi.jl, a
+numerical simulation framework for conservation laws. We visualize the
+results using Julia’s plotting packages. We highlight select features
+in Trixi.jl, like adaptive mesh refinement (AMR) or shock capturing,
+useful for practical applications with complex transient behavior.
+
+
+### More detailed description
+
+Applications of interest in computational fluid mechanics typically occur
+on domains with curved boundaries. Further, the solution of a non-linear
+physical model can develop complex phenomena such as discontinuities,
+singularities, and turbulence.
+
+Attacking such complex flow problems may seem daunting. In this talk,
+however, we present a toolchain with components entirely available in
+the Julia ecosystem to do just that. In broad strokes the workflow is:
+
+1. Use [HOHQMesh.jl](https://github.com/trixi-framework/HOHQMesh.jl)
+   to interactively prototype and visualize a domain with curved boundaries.
+2. HOHQMesh generates an all quadrilateral mesh amenable for high-order numerical
+   methods.
+3. The mesh file is passed to [Trixi.jl](https://github.com/trixi-framework/Trixi.jl),
+   a numerical simulation framework for conservation laws.
+4. Solution-adaptive refinement of the mesh within Trixi is handled by
+   [P4est.jl](https://github.com/trixi-framework/P4est.jl).
+5. After the simulation, a first visualization is made with
+   [Makie.jl](https://makie.juliaplots.org/stable/).
+6. Solution data can also be exported with
+   [Trixi2Vtk.jl](https://github.com/trixi-framework/Trixi2Vtk.jl)
+   for visualization in
+   external software like [ParaView](https://www.paraview.org/).
+
+The strength and simplicity of this workflow is through the combination
+of several packages either originally written in Julia, like Trixi.jl,
+or wrappers, like P4est.jl or HOHQMesh.jl, that provide Julia users access
+to powerful, well-developed numerical libraries and tools written in other
+programming languages.
+
+## Getting started
+
+
+### Installing Julia
+To obtain Julia, go to https://julialang.org/downloads/ and download the latest
+stable release (v1.7.3 as of 2022-06-28; neither use the LTS release nor
+Julia Pro). Then, follow the
```

Comment:
I am just warning people to not use either of these Julia "flavors". Mostly from personal experience, JuliaPro caused a lot of funniness when I first started learning Julia.

Reply:
Can we reword it "for example, Trixi may not work with the JuliaPro distribution or the LTS release" (which I think is version 1.6.3 now?).

Reply:
I read earlier today that JuliaPro was [basically discontinued](https://discourse.julialang.org/t/is-juliapro-discontinued/67670/2). Also because I think Atom is dying. We can still add something like the above to be cautious of it as well as the LTS release.

Reply:
Sounds good. We also note Trixi needs Julia 1.7 so the LTS warning isn't strictly necessary (but might be nice to include).

Reply:
Done I added a short rearranged version of the sentence above that you posted.